### PR TITLE
Remove delegate implimentation, replace with app resume

### PR DESCRIPTION
**Pull request by sitefinitysteve**
- Removed the custom delegate which caused the conflict here https://github.com/chronogolf/nativescript-store-update/issues/21
- Hook into app resume method.  Didn't do launch *as well* since resume is called at launch so implementing both caused a double-check at startup.

### CODE REVIEW

File: src/store-update.ios.ts [hunk 1]
```diff
@@ -26,6 +23,11 @@ export class StoreUpdate {
       ...config,
       onConfirmed: StoreUpdate._openStore.bind(StoreUpdate),
     })
+
+    //Hook into resume event to check for version
+    app.on(app.resumeEvent, function (args) {
+        StoreUpdate.checkForUpdate();
+    });
```
Comment (jeremypele) [inline src/store-update.ios.ts]: Is the `resumeEvent` triggered at launch time ? If not I would implement `launchEvent ` as well

**Comment by sitefinitysteve** [comment]
Yes, this was in my note... So putting both triggers a double callback at
launch

On Dec 13, 2017 1:52 PM, "Jeremy PELE" <notifications@github.com> wrote:

*@jeremypele* requested changes on this pull request.
------------------------------

In src/store-update.ios.ts
<https://github.com/chronogolf/nativescript-store-update/pull/22#discussion_r156748798>
:

> @@ -26,6 +23,11 @@ export class StoreUpdate {
       ...config,
       onConfirmed: StoreUpdate._openStore.bind(StoreUpdate),
     })
+
+    //Hook into resume event to check for version
+    app.on(app.resumeEvent, function (args) {
+        StoreUpdate.checkForUpdate();
+    });

Is the resumeEvent triggered at launch time ? If not I would implement
launchEvent as well

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/chronogolf/nativescript-store-update/pull/22#pullrequestreview-83278441>,
or mute the thread

**Comment by jeremypele** [comment]
Ok right, didn't catch up this one. Thanks for the PR

**Comment by sitefinitysteve** [comment]
Thx guys!

**Pull request merged**